### PR TITLE
Deal with new ffi Ruby version requirement

### PR DESCRIPTION
**Pull request by benoittgt**
https://github.com/ffi/ffi/pull/683

Related:
- https://github.com/ffi/ffi/issues/699

### CODE REVIEW

**Comment by benoittgt** [comment]
CI seems failing because of rspec-mocks ffi build error

File: Gemfile [hunk 1]
```diff
@@ -30,8 +30,10 @@ end
 # allow gems to be installed on older rubies and/or windows
 if RUBY_VERSION < '2.2.0' && !!(RbConfig::CONFIG['host_os'] =~ /cygwin|mswin|mingw|bccwin|wince|emx/)
   gem 'ffi', '< 1.10'
-elsif RUBY_VERSION < '1.9'
-  gem 'ffi', '< 1.9.19' # ffi dropped Ruby 1.8 support in 1.9.19
+elsif RUBY_VERSION < '2.0'
+  # ffi dropped Ruby 1.8 support in 1.9.19
+  # ffi dropped Ruby 1.9 support in 1.11.0
+  gem 'ffi', '< 1.9.19'
```
Comment (JonRowe) [inline Gemfile]: Mistake here, you've not got the split versions.

**Comment by benoittgt** [comment]
I also removed https://github.com/rspec/rspec-expectations/commit/81b60399f624ac42e5e49d73afa687471757f07a because I don't see any error with `bundler audit`. We will see how the CI react.

File: Gemfile [hunk 2]
```diff
@@ -27,13 +27,14 @@ if RUBY_VERSION < '2.0.0' || RUBY_ENGINE == 'java'
   gem 'json', '< 2.0.0' # is a dependency of simplecov
 end
 
+
```
Comment (JonRowe) [inline Gemfile]: Remove this.

**Comment by benoittgt** [comment]
Fail on appveyor seems unrelated.

**Pull request merged**